### PR TITLE
Improve logging classifications

### PR DESCRIPTION
**Pull request by tadaskay**
Fixes: #1290 

- Stdlog is redirected to zerolog
- `go-openvpn` logs are redirected to zerolog
- Restore rolling log file functionality
- As zerolog was introduced in https://github.com/mysteriumnetwork/node/pull/1362 - all `trace` messages now fall under `debug` because zerolog doesn't have trace at the moment. Some levels were also tweaked.

### CODE REVIEW

File: logconfig/config.go [hunk 1]
```diff
@@ -18,88 +18,113 @@
...
 		Logger()
 }
+
+func setGlobalLogger(logger *zerolog.Logger) {
+	log.Logger = *logger
+	stdlog.SetFlags(0)
+	stdlog.SetOutput(log.Logger)
+}
+
+// trimLeftInclusive trims left pat of the string up to and including the prefix
```
Comment (soffokl) [inline logconfig/config.go]:
```suggestion
// trimLeftInclusive trims left part of the string up to and including the prefix
```

File: logconfig/config.go [hunk 2]
```diff
@@ -18,88 +18,113 @@
...
 
-func padRight(s string, length int) string {
-	if len(s) >= length {
-		return s
+// Configure configures logger using app config (console + file, level)
+func Configure(opts *LogOptions) {
+	CurrentLogOptions = *opts
+	log.Info().Msgf("Log level: %s", opts.LogLevel)
+	if opts.Filepath != "" {
+		log.Info().Msg("Log file path: " + opts.Filepath)
```
Comment (anjmao) [inline logconfig/config.go]: Try to be consistant. Use Msgf it for formatted messages.

```go
log.Info().Msgf("Log level: %s", opts.LogLevel) // Ok
log.Info().Msg("Log file path: " + opts.Filepath) 
```

File: logconfig/config.go [hunk 3]
```diff
@@ -18,88 +18,113 @@
...
+		fileWriter, err := fileWriter(opts)
```
Comment (anjmao) [inline logconfig/config.go]: Could be

```go
fileWriter, err := newFileWriter(opts)
```

File: logconfig/config.go [hunk 4]
```diff
@@ -18,88 +18,113 @@
...
+		if err != nil {
+			log.Error().Err(err).Msg("Failed to configure file logger")
+		} else {
+			multiWriter := io.MultiWriter(consoleWriter(), fileWriter)
```
Comment (anjmao) [inline logconfig/config.go]: Could be consoleWriter -> newConsoleWriter

File: logconfig/config.go [hunk 5]
```diff
@@ -18,88 +18,113 @@
...
-//`
+const (
+	timestampFmt = "2006-01-02T15:04:05.000"
+)
 
-// BootstrapWith loads log package into the overall system
-func BootstrapWith(opts *LogOptions) {
-	if opts != nil {
-		CurrentLogOptions = *opts
+// Bootstrap configures logger defaults (console)
```
Comment (anjmao) [inline logconfig/config.go]: Dot at the end.

File: logconfig/config.go [hunk 6]
```diff
@@ -18,88 +18,113 @@
...
-	)
+	openvpn.UseLogger(zerologOpenvpnLogger{})
+	logger := makeLogger(consoleWriter())
+	setGlobalLogger(&logger)
 }
 
-func padRight(s string, length int) string {
-	if len(s) >= length {
-		return s
+// Configure configures logger using app config (console + file, level)
```
Comment (anjmao) [inline logconfig/config.go]: Dot at the end.

File: logconfig/config.go (hunk 2 above)
Comment (tadaskay) [reply logconfig/config.go]: Fixed. Thanks.

**Pull request merged**